### DISCUSSION TITLE
Chat room

### DISCUSSION THREAD

**Discussion by captnfab** [openblack/openblack#411] [Q&A] [answered]
Hi!
Is there a chatroom somewhere (IRC / Matrix / etc.) where one can idle and occasionally chat about openblack?
And if there is none, has anyone a thought about it?

**Comment by bwrsandman** [comment, marked as answer]
We have our discord chat listed on the readme page: [![Discord chat](https://img.shields.io/discord/608729286513262622?logo=discord&logoColor=white)](https://discord.gg/5QTexBU)
